plugins/base/src/main/java/edu/wpi/first/shuffleboard/plugin/base/widget/TextView.java

```diff
@@ -26,23 +27,36 @@
 public class TextView extends SimpleAnnotatedWidget<Object> {
 
   private final StringProperty text = new SimpleStringProperty(this, "text", "");
-  private final StringProperty label = new SimpleStringProperty(this, "label", "");
 
   @FXML
   private Pane root;
+  @FXML
+  private TextField textField;
+  @FXML
+  private NumberField numberField;
+
+  @FXML
+  private void initialize() {
+    text.bind(EasyBind.map(dataProperty(), this::simpleToString));
+    MonadicBinding<Boolean> isNumber = EasyBind.map(dataProperty(), d -> d instanceof Number).orElse(false);
```

Comment:
does this need to be retained for GC reasons?

Reply:
No. This is on the stack so it's gone. I was originally planning in using those on both text fields, but `MonadicBinding<Boolean>`doesn't support .not() like a BooleanBinding